build/Targets/Packages.props

```diff
@@ -3,220 +3,222 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Ah okay.  Looked at teh file and two pieces of feedback:

- Added a blank line, please remove 
- Roslyn.BuildUtil is still referenced and should be removed per earlier comment